src/components/campaigns/InlineDonation.tsx

```diff
@@ -62,6 +63,7 @@ export default function InlineDonation({ campaign }: Props) {
   const summary = campaign.summary.find(() => true)
   const reached = summary ? summary.reachedAmount : 0
   const { data: prices } = useSinglePriceList()
+  const { data: donations, error: donationHistoryError } = useCampaignDonationHistory(campaign.id)
```

Comment:
We might add  a spinner when history is loading

Reply:
Done in https://github.com/podkrepi-bg/frontend/pull/709/commits/f9d6300e738619a13847789b1f5b473401603fb6